ext/msgpack/packer_ext_registry.c

```diff
@@ -18,11 +18,8 @@
 
 #include "packer_ext_registry.h"
 
-static ID s_call;
-
 void msgpack_packer_ext_registry_static_init(void)
 {
-    s_call = rb_intern("call");
 }
```

Comment:
This is now an empty function, can we remove it?

Reply:
👍